deploy/helm/templates/metrics-collector-env-config-map.yaml

```diff
@@ -7,15 +7,6 @@ metadata:
   labels:
 {{ include "common.labels" . | indent 4 }}
 data:
-{{- if .Values.prometheus.enabled }}
-{{- if .Values.prometheus.forceNamespace }}
-  PROMETHEUS_URL: "{{ .Release.Name }}-prometheus-server.{{ .Values.prometheus.forceNamespace }}.svc.cluster.local:80"
-{{- else }}
-  PROMETHEUS_URL: "{{ .Release.Name }}-prometheus-server.{{ .Release.Namespace }}.svc.cluster.local:80"
-{{- end }}
-{{- else }}
-  PROMETHEUS_URL: {{ quote .Values.otel.metrics.prometheus.url }}
```

Comment:
This should remain here. If `.Values.otel.metrics.prometheus.url` is filled with customer's deployed prometheus, we can scrape it. There is functionality in OTEL Collector for it: https://github.com/solarwinds/swi-k8s-opentelemetry-collector/blob/master/deploy/helm/metrics-collector-config.yaml#L1504

Reply:
fixed